.erda/migrations/dicehub/20211220-decehub.sql

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021 Terminus, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ALTER table `dice_release`
+    ADD COLUMN `markdown`                 text NOT NULL DEFAULT '' COMMENT 'Changelog',
+    ADD COLUMN `is_stable`                tinyint(1) NOT NULL DEFAULT 0 COMMENT 'stable表示非临时制品',
```

Comment:
Use type field to describe the release type

Reply:
Start a new pull request to fix it.